.github/workflows/qa.yml

```
@@ -0,0 +1,49 @@
+name: QA
+
+on: [push, pull_request]
+
+env:
+  # black default
+  MAX_LINE_LENGTH: 88
+
+jobs:
+  check-qa:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
```

```suggestion
          python-version: "3.11"
```

I think our images ship with py3.8 but I think we want to default to 3.11 soon